### ISSUE TITLE
ParseGEOJSONObject should be a public API

### ISSUE THREAD

**Issue by Neku** [andreacremaschi/GEOSwift#4]
Hi,
I'm wondering why the only way to feed geojson from a library is through a file.
A common case is to parse part of a JSON response, so it would be great to have a public api that takes a dictionary.

**Comment by andreacremaschi** [comment]
Well, I see your point and I think you are right. The best place to add the feature you ask would be a public func in the Geometry class extension declared in GEOJSON.swift, what do you think?
I will leave the issue open as a remainder and implement it as soon as I can, but I could accept a PR as well :)

**Comment by andreacremaschi** [comment]
hi @Neku this has been addressed in GEOSwift 0.2.1. I added a method with this interface: 

```swift
public class func fromGeoJSONDictionary(dictionary: Dictionary<String, AnyObject>) -> Array<Geometry>?
```

Does it solve your needs?

**Comment by Neku** [comment]
Yes, thank you very much.
